edgar-app/src/app/simulation/confirmation/content.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { useEffect } from 'react';
+import { Text, Box, useToast } from '@chakra-ui/react';
+import Link from 'next/link';
+import { useSearchParams } from 'next/navigation';
+
+import SimulationLayout from 'components/simulationPages/SimulationLayout';
+import SimulationButton from 'components/simulationPages/SimulationButton';
+
+import { useLazyGetPatientAppointmentByIdQuery } from 'services/request/appointments';
+import { useLazyGetDoctorByIdQuery } from 'services/request/doctor';
+
+const SimulationConfirmationContent = (): JSX.Element => {
+	const searchParams = useSearchParams();
+	const appointmentId = searchParams.get('appointmentId');
+	const [triggerAppointment, resultAppointment] = useLazyGetPatientAppointmentByIdQuery();
+	const [triggerDoctor, resultDoctor] = useLazyGetDoctorByIdQuery();
+	const toast = useToast({ duration: 3000, isClosable: true });
+
+	useEffect(() => {
+		if (appointmentId) triggerAppointment(appointmentId);
+	}, [appointmentId]);
+
+	useEffect(() => {
+		if (resultAppointment.data) triggerDoctor(resultAppointment.data.doctorId);
+	});
```

Comment:
⚠️ You need to add a dependencies otherwise the useEffect will run each time the component is updated that could lead to infinite run